lib/resque.rb

```diff
@@ -187,6 +187,46 @@ def enqueue_front
     end
   end
 
+  attr_writer :per_worker_stats
+  def per_worker_stats
+    if defined? @per_worker_stats
+      @per_worker_stats
+    else
+      @per_worker_stats = true
+    end
+  end
+
+  attr_writer :track_starts
+  def track_starts
+    if defined? @track_starts
+      @track_starts
+    else
+      @track_starts = true
+    end
+  end
+
+  attr_writer :queues_in_names
+  def queues_in_names
+    if defined? @queues_in_names
+      @queues_in_names
+    else
+      @queues_in_names = true
+    end
+  end
```

Comment:
FEATURE: `queues_in_names`

Controls whether workers store their list of queues as part of their name, or under a separate `<worker_name>:queues` key in Redis. Set to `false` in the monolith.

Could be PRed to master. Switching the option off in an existing Resque installation would require either a stop-the-world restart of all workers or additional code to read queues from *both* locations temporarily.